releases/4.0.0-beta.2.md

```diff
@@ -0,0 +1,54 @@
+---
+title: TiDB 4.0.0-beta.2 Release Notes
+category: Releases
+---
+
+# TiDB 4.0.0-beta.2 Release Notes
+
+发版日期：2020 年 03 月 13 日
+
+TiDB 版本：4.0.0-beta.2
+
+TiDB Ansible 版本：4.0.0-beta.2
+
+## TiDB 4.0.0-beta.2 兼容性变化
+
++ Tools
+    - TiDB Binlog
+        - 修复 Drainer 配置 `disable-dispatch`、`disable-causality` 时系统直接报错并退出的问题 [#915](https://github.com/pingcap/tidb-binlog/pull/915)
+
+## 新功能
+
++ TiKV
+    - 支持将动态修改配置的结果持久化存储到硬盘 [#6684](https://github.com/tikv/tikv/pull/6684)
+
++ PD
+    - 支持将动态修改配置的结果持久化存储到硬盘 [#2153](https://github.com/pingcap/pd/pull/2153)
+
++ Tools
+    - TiDB Binlog
+        - 新增 TiDB 集群之间数据双向复制功能 [#879](https://github.com/pingcap/tidb-binlog/pull/879) [#903](https://github.com/pingcap/tidb-binlog/pull/903)
+    - TiDB Lightning
+        - 新增配置 TLS 功能 [#40](https://github.com/tikv/importer/pull/40) [#270](https://github.com/pingcap/tidb-lightning/pull/270)
+    - 新增 TiCDC 工具，用于捕捉 TiKV 变化的数据并同步到下游 Kafka、MySQL 协议的数据库；确保数据最终一致性，若下游是 Kafka 可确保行级别的有序；提供进程级别的高可用能力。
```

Comment:
@TomShawn Shall we separate this sentence into several sub-items, just like the English version?
```suggestion
    - 新增 TiCDC 工具，提供以下功能：
        - 捕捉 TiKV 变化的数据，同步到下游 Kafka、MySQL 协议的数据库
        - 确保数据最终一致性，若下游是 Kafka，也可确保行级别的有序
        - 提供进程级别的高可用能力
```

Reply:
I think this is a better way, but you can check with the PM.

Reply:
@scsldb PTAL~